WMFData/Sources/WMFData/Data Controllers/Article Tabs/WMFArticleTabsDataController.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import UIKit
+import CoreData
+
+public class WMFArticleTabsDataController {
+    
+    public let coreDataStore: WMFCoreDataStore
+    private let userDefaultsStore: WMFKeyValueStore?
+    private let developerSettingsDataController: WMFDeveloperSettingsDataControlling
+    
+    public init(coreDataStore: WMFCoreDataStore? = WMFDataEnvironment.current.coreDataStore, userDefaultsStore: WMFKeyValueStore? = WMFDataEnvironment.current.userDefaultsStore, developerSettingsDataController: WMFDeveloperSettingsDataControlling = WMFDeveloperSettingsDataController.shared) throws {
+
+        guard let coreDataStore else {
+            throw WMFDataControllerError.coreDataStoreUnavailable
+        }
+        self.coreDataStore = coreDataStore
+        self.userDefaultsStore = userDefaultsStore
+        self.developerSettingsDataController = developerSettingsDataController
+    }
+    
+    // MARK: Entry point
+
+   public var shouldShowArticleTabs: Bool {
+        get {
```

Comment:
For this, I think you can just do `return developerSettingsDataController. enableArticleTabs` instead of digging into user defaults here. Sorry I missed this before approval. I can fix it in my next PR.